with-tinybase/App.js

```diff
@@ -2,27 +2,27 @@ import { useState } from 'react';
 import Constants from 'expo-constants';
 import * as SQLite from 'expo-sqlite';
 import {
+  FlatList,
   Platform,
-  ScrollView,
+  SafeAreaView,
```

Comment:
here's a funny thing about safe areas and react-native.. it's best to not use the SafeAreaView provided in react-native core. [we recommend using react-native-sare-area-context](https://docs.expo.dev/develop/user-interface/safe-areas/), which behaves better in various circumstances and also provides a nice hook for accessing the insets (which you can use to replace the `Constants.statusBarHeight` usage)